openssl-dynamic/src/main/c/ssl.c

```diff
@@ -2086,6 +2090,115 @@ TCN_IMPLEMENT_CALL(void, SSL, setTlsExtHostName)(TCN_STDARGS, jlong ssl, jstring
     TCN_FREE_CSTRING(hostname);
 }
 
+TCN_IMPLEMENT_CALL(jobjectArray, SSL, authenticationMethods)(TCN_STDARGS, jlong ssl) {
+    SSL *ssl_ = J2P(ssl, SSL *);
+    const STACK_OF(SSL_CIPHER) *ciphers = NULL;
+    int len;
+    int i;
+    jobjectArray array;
+
+    if (ssl_ == NULL) {
```

Comment:
we use `TCN_ASSERT(ssl_ != NULL)` in other methods ... lets be consistent and choose one